packages/react-static/src/static/components/HeadWithMeta.js

```diff
@@ -87,6 +91,7 @@ export default async function makeHeadWithMeta(state) {
         <meta name="generator" content="React Static" />
         {head.base}
         {useHelmetTitle && head.title}
+        {childrenMeta}
```

Comment:
`children` comes from the `DocumentTemplate` here - hence, are site wide. Figured site wide meta had priority over per-page meta.